src/Controller/HomeController.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace App\Controller;
+
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\Routing\Annotation\Route;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Template;
+use Symfony\Component\HttpKernel\Exception\BadRequestHttpException;
+use Symfony\Component\PropertyAccess\PropertyAccess;
+use Symfony\Component\Security\Core\User\UserInterface;
+use Symfony\Component\Security\Core\Authorization\AuthorizationCheckerInterface;
+use Symfony\Component\Security\Core\Exception\AccessDeniedException;
+
+class HomeController extends BaseController
+{
+
+    /**
+     * @Route("/",
+     *      name="home",
+     *     )
+     * @Template("home.html.twig")
+     *
+     */
+    public function home()
+    {
+        return [];
+    }
```

Comment:
@daria-sieroshtan 
are you sure you want to have an empty home screen? Few options:
- Put core functionality there, like "run a workout"
- Show some stats
It's not mandatory to do it today